common/logisticspipes/gui/GuiGuideBook.java

```diff
@@ -188,4 +229,12 @@ protected void mouseReleased(int mouseX, int mouseY, int state) {
 		this.dragging = false;
 		super.mouseReleased(mouseX, mouseY, state);
 	}
+
+	@Override
+	public void onGuiClosed() {
+		NBTTagCompound nbtTagCompound = book.getTagCompound();
+		nbtTagCompound.setInteger("page", page);
+		nbtTagCompound.setFloat("sliderProgress", sliderProgress);
```

Comment:
This won't work in anything but creative mode since the client can't modify item data. You'll probably have to write some kind of netcode (packets) to tell the server to change the item data

Reply:
Ok I'll have a look at that

Reply:
Fixed in the latest commit